action-server/db/assessment.py

```diff
@@ -0,0 +1,28 @@
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, func
+from sqlalchemy.dialects.postgresql import JSONB
+
+from .base import Base
+
+# TODO: We might want to use auto-mapping instead of re-defining the actual tables.
```

Comment:
We have a fair share of duplication between this and the alembic table initialisation code. Would this suggestion remove the duplication? Else, any way we could share something between this and alembic?

Reply:
Yes, auto-mapping would solve the issue with duplication here. Since we don't know much about auto-mapping for now, @fparga and I though we could just start with this and move to auto-mapping later.

Reply:
To clarify my thoughts: the biggest concern I have about using the auto-mapper is that we do not have (as of now) a good hook to initialize the auto-mapper other then creating a fake rasa action that would be automatically load at the start. This is kind of hackish...

The other concern I have is that, on the long run, I don't know how we will manage to understand the current schema of the database using only the migration files. (unless there's a way to generate comme king of final schema/documentation from all the migration files)

Reply:
If we don't mind about using an action to initialize the auto-mapper, and not having our schema documented with anything else then alembic migrations scripts, I can change all of this and move right now to the auto-mapper.

@dmorand @fparga  thoughts?

Reply:
IMO using actions for initialization sucks

Reply:
I agree, it's better if we can avoid it.